hipogram/posts/urls.py

```diff
@@ -9,7 +9,8 @@
     path("", PostListView.as_view(), name="list"),
     #path("", ShareView.as_view(), name="share"),
     path('share/', views.post_new, name='post_new'),
-    path('update/', views.delete_post, name='delete_post'),
-    path('update/', views.update_post, name='update_post'),
+    path('update/<int:id>/', views.update_post, name='update_post'),
+    path('update/<int:id>/', views.delete_post, name='delete_post'),
+    path('listfiltered/', views.post_filter_list_view, name='list_filtered'),
```

Comment:
We decided to wait for this one, and also it doesn't related to current PR, please remove this.